test/e2e/features/cat-c/10-pass_with_no_faults.feature

```diff
@@ -15,7 +15,7 @@ Feature: A Driving Examiner Completes a Passed User Journey in Category C
       When I open the reversing diagram
       Then I should see the reversing diagram modal
       And I close the reversing diagram modal
-      Then I close the revresing diagram drop down
+      Then I close the reversing diagram drop down
```

Comment:
Change this line on every page it is used.

Reply:
done